### ISSUE TITLE
LHR-TUNE can't keep it up

### ISSUE THREAD

**Issue by Aurum420** [trexminer/T-Rex#687]
Started with --lhr-tune 36 and it detected a lock and stepped down 1% and tried again. This is during dual-mining with ETH+RVN on a 3080 Ti . It eventually ends up at LHR 10 and the hashrate is so slow it can't even submit a share. If no other program demands time on the GPU the ETH hashrate will settle in around LHR30ish. If another program wants to use the GPU then it keeps dropping until it flatlines at LHR 10. The RVN hashrate seems to behave okay. Looks like this dual-mining LHR-TUNE is only suitable for dedicated mining rigs.

I've only yet to see LHR-TUNE go up, just down. It would be better if another program wanted the GPU if the ETH hashing just paused until it got full control of the GPU back. RVN would just keep running. Maybe make LHR-TUNE seek a target or oscillate within a user-defined range.

This is on an EVGA GeForce RTX 3080 Ti FTW3 ULTRA GAMING 12G-P5-3967-KR. The EVGA nomenclature says the -KR cards are NOT LHR. They lied to their customers because it is LHR.

**Comment by heliosgnosis** [comment]
Yeah but he is correct Theangelofspace15 when I bought all of my EVGA cards KR versions and this was  in the beginning and mid to end of may there was ZIP NADA nothing saying LHR BEWARE anywhere on the site or in the description warning me of buying a broken on purpose product at over MSRP from the manufacture none the less ,  when all the ETH mining is done all of us ETH miners who got tricked into buying these "investments" that were not listed as broken and sue NVIDIA and the companies that supported their bullshit to get sales of their mining cards rolling out strong by targeting one ALGO only and tens of millions bought these units unaware of them so being broken,  this is illegal in many countries across the world to do especially since conflict of interest on NVIDIA's side is easy to prove,   make back the loses for 7 months plus ETH price gain loses,   yeah that would fix nvidia's problem with thinking they can do as they like ,  like this is some 10th century Chinese mining operation in some poor African nation and wild wild west rules apply to give them full power to do as they want with not penalty . Draining 2 quarters of profits from them would set them back to reality and make future victims never happen again. The forgot the consumer matters not use the consumer to make more profits like the did and it can be proven they did it and their partners without warning the consumer of what they were buying while buying it till into June BTW for EVGA,  that is illegal and anyones loses EVGA and NVIDIA so owe to tens of millions

**Comment by Theangelofspace15** [comment]
Im pretty sire the LHR announcement was made BEFORE the 3080TI release. Thats why we all know that ALL 3080TI are LHR. But well.

**Comment by draft91** [comment]
Same card same issue. With msi 3070ti gaming trio im running eth+erg without much problem. After 12-15 hours of working its lhr drops to 27-28. But with evga 3080ti eth+rvn is going down to lhr 10 like a hell. Oc settings 0 core +1000 mem PL 75%
